src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -1327,39 +1332,33 @@ public void test_fetch_with_prune() throws Exception {
         ObjectId newAreaHead = newArea.head();
         Set<Branch> remoteBranches = newArea.git.getRemoteBranches();
         assertBranchesExist(remoteBranches, "origin/master", "origin/branch1", "origin/branch2", "origin/HEAD");
-        assertEquals("Wrong count in " + remoteBranches, 4, remoteBranches.size());
+        assertBranchCount(remoteBranches, 4);
```

Comment:
I have seen this pattern (check for branch names + branch count) here and in some other tests.
Currently it always requires 2 assertions where just 1 fact should be asserted - that the branches match exactly independent of their order.
Hamcrest's [containsInAnyOrder(...)](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#containsInAnyOrder(T...)) is precisely for this purpose.